packages/fields/src/fields/title/title-view.tsx

```diff
@@ -0,0 +1,62 @@
+/**
+ * WordPress dependencies
+ */
+import { __experimentalHStack as HStack } from '@wordpress/components';
+import { decodeEntities } from '@wordpress/html-entities';
+import { __ } from '@wordpress/i18n';
+import { useSelect } from '@wordpress/data';
+import { store as coreStore } from '@wordpress/core-data';
+import type { Settings } from '@wordpress/core-data';
+
+/**
+ * Internal dependencies
+ */
+import type { BasePost } from '../../types';
+import { getItemTitle } from '../../actions/utils';
+
+const TitleView = ( { item }: { item: BasePost } ) => {
+	const { frontPageId, postsPageId } = useSelect( ( select ) => {
+		const { getEntityRecord } = select( coreStore );
+		const siteSettings: Settings | undefined = getEntityRecord(
+			'root',
+			'site',
+			''
```

Comment:
I just realised this works because the above PR makes `key` [optional](https://github.com/WordPress/gutenberg/pull/65426/files#diff-c9806943cd2157856848ecea1d5d7f97f6bd462827ed87b246be86c758f684f8R313) in `getEntityRecord`. Maybe this isn't ideal, although I'm not sure the best way around it.

Reply:
Oh, I see. Thanks for providing a fix :)